src/debuginfo.rs

```diff
@@ -89,15 +238,47 @@ impl<'gcc, 'tcx> DebugInfoMethods<'tcx> for CodegenCx<'gcc, 'tcx> {
         _fn_abi: &FnAbi<'tcx, Ty<'tcx>>,
         _maybe_definition_llfn: Option<RValue<'gcc>>,
     ) -> Self::DIScope {
-        unimplemented!();
+        //unimplemented!();
     }
 
     fn dbg_loc(
         &self,
         _scope: Self::DIScope,
         _inlined_at: Option<Self::DILocation>,
-        _span: Span,
+        span: Span,
     ) -> Self::DILocation {
-        unimplemented!();
+        //unimplemented!();
+        let pos = span.lo();
```

Comment:
Did you write this whole function or was it copied from the LLVM codegen? If it was copied, could you please tell me where it comes from?

Reply:
The idea was from LLVM, but many data structures are not present in GCC. cg_llvm used a one(2?)-liner which is a simple call to the I guess this is how it should work and it turned out to be of some effect.